.typespec/streaming/models.tsp

```diff
@@ -44,6 +44,9 @@ union AssistantStreamEvent {
 
 union ThreadStreamEvent {
   {
+    /** Whether to enable input audio transcription. */
+    enabled?: boolean,
```

Comment:
Is this new property reflected anywhere in the library?

Reply:
Not yet -- haven't investigated the Assistants changes and it looks suspicious to me. Filed a work item to track but I don't think we need to update the public surface yet.